### ISSUE TITLE
Creating new migration not working

### ISSUE THREAD

**Issue by Fosol** [dotnet/efcore#22762]
I'm working on an open-source project, however migrations are not working.  Essentially every time I create a migration it creates a full migration containing all existing tables.  This is a code-first database, it did not pre-exist.

open source project - https://github.com/bcgov/pims

The EF Core project is in the following folder `/backend/Dal`

Currently there is a initial migration named `Initial`.  This is essentially version 1.

I run the command `dotnet ef migrations add v01.00.01` and it creates a new migration.

**Expected Result**
A new blank migration containing no changes.

If I made changes to entities, then it should only include those changes.

**Actual Result**
The migration includes all the objects already created in the **Initial** migration.  

EF Core version: 3.1.5
Database provider: Microsoft.EntityFrameworkCore.SqlServer
Target framework: netcoreapp3.1
Operating system: Windows 10

**Comment by ajcvickers** [comment]
@Fosol I am not able to reproduce this. With your code, I already see a migration with that name:

```
PS C:\Stuff\Repros\PIMS-dev\PIMS-dev\backend\dal> dotnet ef migrations list
Build started...
Build succeeded.
20200722193504_Initial
20200925194028_v01.00.01
```

If I add another, it is empty. If I remove and add a new migration I get changes, but this is because the model has differences from the initial migration. Adding another after this is again empty.

**Comment by Fosol** [comment]
We have solved the issue.  The project file was excluding the migration files.  Not sure how that occurred, but I suspect a bad merge or dev accident. 

I've removed those lines from the project file and migrations have stared working again.

I think this just highlights something inconsistent with how the CLI behaves.  We could update the DB with the migration, but we couldn't create another new migration.
